src/components/JunoToggleSwitch.tsx

```
@@ -6,94 +1,101 @@
-          {/* Switch Handle */}
+      {tooltip ? <Tooltip label={tooltip}>{titleNode}</Tooltip> : titleNode}
+      <Group gap={0}>
+        <Stack justify="center" gap={trackPadding} py={trackPadding}>
+          {data.map((d) => (
+            <Title
+              onClick={() => onSelect(d.value)}
+              order={6}
+              h={switchHeight}
+              style={{ cursor: "pointer", lineHeight: "1.4rem" }}
+              pr="sm"
+            >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)